doc/contributor/using-eclipse.md

```diff
@@ -41,9 +43,32 @@ There should be now 4 projects in your workspace:
 * `TRUFFLERUBY-TEST.dist`
 * `TRUFFLERUBY.dist`
 
-## Running from the Eclipse Files Directly
+### Running from the Eclipse Files Directly
```

Comment:
FYI I think this no longer works (without what you added below)

Reply:
Moved that around to avoid confusion.